src/Parallel/Serialize.hpp

```diff
@@ -0,0 +1,54 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the serialize and deserialize functions.
+
+#pragma once
+
+#include <vector>
+
+#include <pup.h>
+
+/*!
+ * \ingroup Parallel
+ * \brief Serialize an object using PUP.
+ *
+ * The type to serialize as must be explicitly specified.  We require
+ * this because a mismatch between the serialize and deserialize calls
+ * causes undefined behavior and we do not want this to depend on
+ * inferred types for safety.
+ *
+ * \tparam T type to serialize to
+ */
```

Comment:
Use either "type to serialize"   or "type to serialize as"  in both the long description and the template parameter description